### ISSUE TITLE
Separate library and CLI

### ISSUE THREAD

**Issue by elnappo** [toolswatch/vFeed#64]
At the moment the CLI calls the library which prints the results. This makes it hard to use the library as an API in third-party code. I.e. return the results from a function call, instead of printing the results out and let the vfeedcli handle the front-end.
This would allow easy integration of your work intro third-party programs.

**Comment by toolswatch** [comment]
hi

can you elaborate a little by giving a concrete example. If I'm.not mistaken you can still use vFeed as library without any interaction with the CLI.
see api_calls.py example + wiki documentation

by the way thanks so much for your valuable contributions ... I will try to address them ASAP. 

I will release a new API this weekend (but not sure if I can integrate some of your commit and issues...at least for this release). But I definitely added all.of them.to my next coming one

**Comment by elnappo** [comment]
Your'e right, I can interact with the vFeed library in my code without the CLI. But the problem with this is that all results from e.g. a search are printed out to stdout within the library.

```python
# bad library
def search(query):
    results = do_the_search(query)
    for result in results:
        print(result) # frontend code in backend/library

# bad CLI part
search("test")

# good library
def search(query):
    results = do_the_search(query)
    return results
  
# good CLI part
for result in search("test"):
    print(result)
```

In my case I want to further process the results from the search. If the results are printed out from the library the code is not reusable because frontend code (display data to the user) is part of the library/backend code. Furthermore is it not a good idea to call `sys.exit()` in a library, if I integrate your library into my code and no results are found my program gets terminated by your library, instead raise a exception.
Thank you for your work done so far!

**Comment by toolswatch** [comment]
By the way I had this in my radar actually. I will fix it to return JSON
instead of printing ...

Thanks so much !!

**Comment by toolswatch** [comment]
And I'm open for any code enhancement and tricks as I'm not a developer. I
learned quickly python to express this vFeed idea ;)

**Comment by toolswatch** [comment]
Fixed with new release 0.7
